chalk-ir/src/lib.rs

```diff
@@ -110,7 +110,7 @@ pub enum TypeName {
 /// `forall<T> { Goal(T) }` (syntatical representation)
 /// `forall { Goal(?0) }` (used a DeBruijn index)
 /// `Goal(!U1)` (the quantifier was moved to the environment and replaced with a universe index)
-/// See https://rust-lang-nursery.github.io/rustc-guide/mir/regionck.html#skol for more.
+/// See https://rust-lang.github.io/rustc-guide/borrow_check/region_inference.html#placeholders-and-universes for more.
```

Comment:
* mir/regionck -> borrow_check/region_inference in https://github.com/rust-lang/rustc-guide/pull/190/files#diff-fcc56c02ee24c620b336bed9bf0d5417
* skol -> placeholder in https://github.com/rust-lang/rustc-guide/pull/217/files#diff-fcc56c02ee24c620b336bed9bf0d5417
* placeholder -> placeholders-and-universes in https://github.com/rust-lang/rustc-guide/pull/234/files#diff-fcc56c02ee24c620b336bed9bf0d5417